src/main/java/executor/service/config/ThreadPoolConfigFactory.java

```diff
@@ -0,0 +1,30 @@
+package executor.service.config;
+
+import executor.service.config.properties.PropertiesConfig;
+import executor.service.model.ThreadPoolConfig;
+
+import static executor.service.config.properties.PropertiesConstants.*;
+
+public class ThreadPoolConfigFactory {
```

Comment:
This code should be part of the DI.

![image](https://github.com/GFLCourses5/worker/assets/140727960/50834a9f-0a1f-468e-8288-bd8206cd2d3d)

Reply:
thanks, I have access to Trello too